frontend/src/app/shared/helpers/svgFileContentFetcher.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+import { of } from 'rxjs';
+import { catchError, switchMap } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class SvgFileContentFetcher {
+    // eslint-disable-next-line no-empty-function
+    constructor(private http: HttpClient, private sanitizer: DomSanitizer) {}
+
+    public fetchSvgContent(svgFilePath: string) {
+        return this.http.get(svgFilePath, { responseType: 'text' }).pipe(
+            switchMap((response) => of(this.sanitizer.bypassSecurityTrustHtml(response))),
+            catchError((error) => {
+                console.error('Error fetching SVG content:', error);
+
+                return of(null);
+            }),
+        );
```

Comment:
Is it so difficult to fetch svg icon?

Reply:
because I thought that icon should be dynamic I'm fetching it via this method. so should I move svg icon to the html so it will always be for the branches combobox?